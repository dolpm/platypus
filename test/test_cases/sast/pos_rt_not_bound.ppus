pipe test |> [] |> [x: int] |> unit {
  |> ();
}

pipe main |> [] |> [] |> unit {
  test <| [1];
  |> ();
}