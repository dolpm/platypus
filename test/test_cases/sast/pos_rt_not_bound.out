pipe test1 |> ['a] |> [x: &'a int] |> &'a int {
{(id: test1_with_args)
{(id: test1_wrapper)
  |> (&'a int : x);
}
}
}

pipe test |> [] |> [x: int] |> unit {
{(id: test_with_args)
{(id: test_wrapper)
  |> (unit : ());
}
}
}

pipe main |> [] |> [] |> unit {
{(id: main_with_args)
{(id: main_wrapper)
  (unit : test <| [(int : 1)]);
  int x <| (int : 1);
  (&'_ int : test1 <| [(&'_ int : &(int : x))]);
  |> (unit : ());
}
}
}
